src/ResourceManager/TrafficManager/Commands.TrafficManager2/help/Add-AzureRmTrafficManagerEndpointConfig.md

```diff
@@ -105,7 +105,7 @@ Accept wildcard characters: False
 ```
 
 ### -GeoMapping
-The list of regions mapped to this endpoint when using the 'Geographic' traffic routing method. Please consult Traffic Manager documentation for a full list of accepted values.
+The list of regions mapped to this endpoint when using the 'Geographic' traffic routing method. Please consult Traffic Manager documentation for a [full list of accepted values] (https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions).
```

Comment:
@dilukose the space between the ending square bracket and opening parenthesis is causing the markdown to not get rendered correctly:

[full list of accepted values] (https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions)

vs.

[full list of accepted values](https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions)